data/Dockerfiles/php-fpm/Dockerfile

```diff
@@ -12,9 +12,28 @@ RUN set -x \
 \
 # Install Dependencies 
 && apk add --update \
-&& apk add --no-cache bash curl redis mariadb-client libressl php7-fpm \
-php7-common php7-intl php7-pdo php7-pdo_mysql php7-xmlrpc php7-redis php7-openssl php7-json php7-curl php7-ctype \
-php7-pear php7-pear-auth_sasl php7-pear-net_smtp php7-pear-net_idna2 php7-pear-mail_mime \
+&& apk add --no-cache \
+  bash \
+  curl \
+  redis \
+  mariadb-client \
+  libressl \
+  php7-fpm \
+  php7-common \
+  php7-intl \
+  php7-pdo \
+  php7-pdo_mysql \
+  php7-xmlrpc \
+  php7-redis \
+  php7-openssl \
+  php7-json \
+  php7-curl \
+  php7-ctype \
+  php7-pear \
+  php7-pear-auth_sasl \
+  php7-pear-net_smtp \
+  php7-pear-net_idna2 \
+  php7-pear-mail_mime \
```

Comment:
Well, I have to emphasize the "sorted". Sorry for bugging you with this, but only then the rule makes sense.